third_party/rust/aes/v0_8/BUILD.gn

```diff
@@ -54,7 +54,10 @@ cargo_crate("lib") {
     "//brave/third_party/rust/cfg_if/v1:lib",
     "//brave/third_party/rust/cipher/v0_4:lib",
   ]
-  if (target_cpu == "x64" || target_cpu == "x86") {
+  if (target_cpu == "arm64" || target_cpu == "x64" || target_cpu == "x86") {
     deps += [ "//brave/third_party/rust/cpufeatures/v0_2:lib" ]
   }
+  if (target_os == "android" && target_cpu == "arm64") {
```

Comment:
aes Cargo.toml doesn't specify this, what was the error?

Reply:
cc: @cdesouza-chromium

Reply:
We were getting the following error whenever `cpufeatures` was included on the `android`/`arm64` build:

```
compile_error!("This crate works only on `aarch64`, `loongarch64`, `x86`, and `x86-64` targets.");
```